playbooks/validation.yml

```diff
@@ -51,6 +46,9 @@
 
 # run some simple slurm jobs
 - hosts: login
+  become: true
+  become_user: "{{ jasmin_users | map(attribute='username') | first }}"
```

Comment:
I hope the first user doesn't mind being used, these jobs might be accounted for somewhere :-)

Reply:
I suppose we'll want to run the validation playbook only on short-lived test clusters.